### PR TITLE
 Generate 'migrate' for prisma using mongo

### PR DESCRIPTION
**Pull request by matheusmfeitoza**
Added a minor content for prisma users with mongo DB to generate schemas and changes.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/recipes/prisma.md [hunk 1]
```diff
@@ -181,6 +181,11 @@ With your Prisma models in place, you can generate your SQL migration files and
 ```bash
 $ npx prisma migrate dev --name init
 ```
+OR for prisma using mongodb
+
+```bash
+$ prisma db push
```
Comment (kamilmysliwiec) [inline content/recipes/prisma.md:187]: npx?

File: content/recipes/prisma.md [hunk 2]
```diff
@@ -181,6 +181,11 @@ With your Prisma models in place, you can generate your SQL migration files and
 ```bash
 $ npx prisma migrate dev --name init
 ```
+OR for prisma using mongodb
```
Comment (kamilmysliwiec) [inline content/recipes/prisma.md:184]:
```suggestion
OR when MongoDB is used:
```

(?)

File: content/recipes/prisma.md (hunk 1 above)
Comment (matheusmfeitoza) [reply content/recipes/prisma.md:187]: When user have prismaCLI instaled can use directly command prisma {command} {argument}

File: content/recipes/prisma.md (hunk 2 above)
Comment (matheusmfeitoza) [reply content/recipes/prisma.md:184]: when MongoDB is used, its better... thnks

File: content/recipes/prisma.md (hunk 1 above)
Comment (kamilmysliwiec) [reply content/recipes/prisma.md:187]: I know but in the other code snippets (in this chapter) we use npx

Comment (matheusmfeitoza) [reply content/recipes/prisma.md:187]: Ok, i'll revise that and submit again. Thanks !

**Pull request closed without merging**